tests/components/zha/conftest.py

```diff
@@ -63,8 +64,6 @@ def globally_load_quirks():
     run.
     """
 
-    import zhaquirks
```

Comment:
I wonder if this one was made/done for performance reasons...

Reply:
I find this confusing, as the fixture is `autouse=True`, but I've changed it to be a pylint disable instead.